crates/teloxide/tests/command.rs

```diff
@@ -236,10 +236,57 @@ fn descriptions_off() {
         Start,
         #[command(hide)]
         Username,
```

Comment:
would be nice to also check `#[description = "off"]`, so that we are mindful of breaking it (i.e. have `Username` use that)